source/02-EV/03.ptx

```diff
@@ -781,6 +805,79 @@ Let <m>W</m> be a subspace of a vector space <m>V</m>.  How are <m>\vspan W</m>
 </fact>-->
     </subsection> 
 
+    <subsection>
+        <title>Cool Down</title>
+        <statement>
+            <p>
+                Before next class, find some time to work on the following.
+            </p>
+        </statement>
```

Comment:
```suggestion
```

For sure the `<statement>` and `</statement>` need to go.

But in general I think we should remove the "before next class" bit. I think these activities will be used in a variety of contexts, which we can suggest in an instructor drive, but to make them versitile we shouldn't hard-specify how a student might be engaging with them.

Reply:
(if it's not clear, committing the suggestion above will remove lines 810-814.)